jujugui/static/gui/src/app/app.js

```diff
@@ -660,6 +660,12 @@ YUI.add('juju-gui', function(Y) {
             this.controllerAPI.connect();
           }
         }
+        // If we have a jimmURL then we need to rely on the controllerAPI
```

Comment:
I am fixing the fact that the controller connection is not made on sandbox in my current branch.